prov/sm2/src/sm2_progress.c

```diff
@@ -42,6 +42,58 @@
 #include "sm2.h"
 #include "sm2_fifo.h"
 
+static inline int sm2_cma_loop(pid_t pid, struct iovec *local,
+			       unsigned long local_cnt, struct iovec *remote,
+			       unsigned long remote_cnt, unsigned long flags,
```

Comment:
I know you're just copying code I wrote but let's drop the flags argument which is always 0.

Reply:
done